app/controllers/curator/metastreams/workflows_controller.rb

```diff
@@ -24,7 +28,7 @@ def set_workflow
     def workflow_params
       case params[:action]
       when 'update'
-        params.require(:administrative).permit!
+        params.require(:workflow).permit(:publishing_state)
```

Comment:
We want to allow `processing_state` as well here.

Reply:
Will change that